backend/routes/news.js

```diff
@@ -17,29 +40,29 @@ router.get("/", async (req, res) => {
   }
 });
 
-// user does not access this
+// user does not access this, for testing purposes must run manually
 router.post("/seed-news", async (req, res) => {
   const newsData = [];
   const apiToken = process.env.API_TOKEN;
 
-  let pageCount = 31; // manually changing for now (TO-DO: automate function to fetch everyday)
+  let pageCount = 1; 
 
-  for (let i = 0; i < 30; i++) {
-    // in order to get 90 total articles at once, figure out how to keep this automatically running every day, for now manually add every day
+  for (let i = 0; i < 40; i++) {
+    // in order to get 120 total articles at once (3 articles per loop)
     const response = await fetch(
       ` https://api.thenewsapi.com/v1/news/top?locale=us&api_token=${apiToken}&language=en&page=${pageCount}`,
     );
     const data = await response.json();
     const articles = data.data;
 
     for (const article of articles) {
+      const publishedDate = new Date(article.published_at);
       const newArticle = {
         name: article.title,
-        releaseDate: article.published_at.toDateString(),
         category: article.categories,
         articleURL: article.url,
         imageURL: article.image_url === "" ? null : article.image_url,
-        userId: req.session.userId,
+        releasedAt:publishedDate,
```

Comment:
What IDE are you using? Wondering if there's a linter you can use to catch style issues like this, or even auto-format on save